source/patterns/@aws-solutions-constructs/core/lib/glue-job-helper.ts

```diff
@@ -143,57 +165,57 @@ export function deployGlueJob(scope: Construct, glueJobProps: glue.CfnJobProps,
     ]
   });
 
-  let _jobRole: IRole;
+  let jobRole: IRole;
```

Comment:
If you prefer to define jobRole as const

`const jobRole = glueJobProps.role ? Role.fromRoleArn(scope, 'JobRole', glueJobProps.role) : defaults.createGlueJobRole(scope);`

Reply:
OK, but formatting differently for clarity